src/ui/flutter_app/lib/shared/theme/app_theme.dart

```diff
@@ -111,36 +73,111 @@ class AppTheme {
     valueIndicatorTextStyle: TextStyle(fontSize: 24),
   );
 
+  static const _cardTheme = CardTheme(
+    margin: EdgeInsets.symmetric(vertical: 4.0),
+    color: _cardColor,
+  );
+
+  static final _textTheme = TextTheme(
+    bodyText1: TextStyle(
+      fontSize: LocalDatabaseService.display.fontSize,
+    ),
+  );
+
   static TextStyle simpleDialogOptionTextStyle = TextStyle(
     fontSize: LocalDatabaseService.display.fontSize + 4.0,
-    color: AppTheme.simpleDialogOptionTextColor,
+    color: _simpleDialogOptionTextColor,
   );
 
   static TextStyle moveHistoryTextStyle = TextStyle(
     fontSize: LocalDatabaseService.display.fontSize + 2.0,
     height: 1.5,
-    color: moveHistoryTextColor,
+    color: _moveHistoryTextColor,
   );
 
-  static double boardTop = isLargeScreen ? 75.0 : 36.0;
-  static double boardMargin = 10.0;
-  static double boardScreenPaddingH = 10.0;
-  static double boardBorderRadius = 5.0;
-  static double boardPadding = 5.0;
+  static TextStyle drawerHeaderTextStyle = TextStyle(
+    fontSize: LocalDatabaseService.display.fontSize + 16,
+    fontWeight: FontWeight.w600,
+  );
 
-  static TextStyle settingsHeaderStyle = TextStyle(
-    color: settingsHeaderTextColor,
+  static TextStyle dialogTitleTextStyle = TextStyle(
     fontSize: LocalDatabaseService.display.fontSize + 4,
+    color: _appPrimaryColor,
   );
 
-  static TextStyle settingsTextStyle =
-      TextStyle(fontSize: LocalDatabaseService.display.fontSize);
+  static const TextStyle copyrightTextStyle = TextStyle(
+    fontSize: _copyrightFontSize,
+  );
 
-  static const cardMargin = EdgeInsets.symmetric(vertical: 4.0);
+  static final TextStyle notationTextStyle = TextStyle(
+    fontSize: 20,
+    color: LocalDatabaseService.colorSettings.boardLineColor,
+  );
 
-  static const double drawerWidth = 250.0;
```

Comment:
TODO: A lot of this code I don’t know where it was moved to?

The bug that the drawer text becomes smaller may be introduced in the process of moving this code.

I'm not sure if there are other things missing some thoughts on the move.

Reply:
this was moved into the drawer widget file with #347. The bug is unlikely to be caused by this it is probably something with this commit indeed. I'll check later.